2010-09-07  Nicolas Setton  <setton@adacore.com>

	* configure: allow deactivating atsui font handling from the
	parameters passed to configure.
	Patch directly imported from MacPorts.

--- configure.orig	2009-07-21 06:59:52.000000000 +1000
+++ configure	2009-10-22 06:14:52.000000000 +1100
@@ -19806,12 +19806,14 @@
 $as_echo "$ac_cv_header_Carbon_Carbon_h" >&6; }
 
 fi
-if test "x$ac_cv_header_Carbon_Carbon_h" = x""yes; then
-  have_atsui=true
-else
-  have_atsui=false
-fi
 
+if test "x$have_atsui" = "x"; then
+  if test "x$ac_cv_header_Carbon_Carbon_h" = x""yes; then
+    have_atsui=true
+  else
+    have_atsui=false
+  fi
+fi
 
 
 #
